guides/integrate-nextauth.md

```diff
@@ -0,0 +1,296 @@
+1. For `BeyondIdentity` _issuer_, copy and paste the value from Applications -> Your New Application -> External Protocol -> Issuer
+
+## Create an Identity and generate a Universal Passkey
+
+Once you have an application in the admin console you are ready to provision users in your realm's directory, generate passkeys, and handle those passkeys in your application.
+
+### Create an Identity
+
+Creating a user can be done either in the admin console or through an API. This guide will use the admin console. Navigate to your realm in the console and click **Identities**, then click **Add identity**.
+
+For more information visit [Workflows: User and Group Provisioning](/docs/v1/workflows/user-provisioning).
+
+<Arcade clip={Clip.CreateIdentity} />
+
+### Generate a passkey
+
+Once you have an identity you can generate a passkey. This step can also be done either in the admin console or through an API. This guide will use the admin console. Navigate back to **Identities** and select the identity you would like to bind to a passkey. Click **Add a passkey**, select your app and the click **Proceed & send email**. The user will receive an enrollment email which they can tap on to bind a passkey to their device.
+
+For more information visit [Workflows: Bind Passkey To User](/docs/v1/workflows/bind-passkey).
+
+<Arcade clip={Clip.CreatePasskey} />
+
+### Bind passkey to device
+
+Once the user taps on the enrollment email, they will be redirected to your application. Intercept the link from the enrollment email. The link that is redirected to your application will take on the following form. A `/bind` path will be appended to your Invoke URL (configured in your application above) as well as several other query parameters.
+
+```
+$invoke_url/bind?api_base_url=<api_base_url>&tenant_id=<tenant_id>&realm_id=<realm_id>&identity_id=<identity_id>&job_id=<job_id>&token=<token>
+```
+
+Once you receive the incoming URL, pass it into the SDK to complete the binding process. You can validate the incoming URL with `isBindPasskeyUrl`. Upon success, a private key will have been created in the device's hardware trust module and the corresponding public key will have been sent to the Beyond Identity Cloud. At this point the user has a passkey enrolled on this device.
+
+```javascript
+if (embedded.isBindPasskeyUrl(url)) {
+  const bindPasskeyResponse = await embedded.bindPasskey(url);
+}
+```
+
+## Authenticate with Passkey
+
+Once you have one passkey bound to a device, you can use it to authenticate. For more information visit [Workflows: Authenticate with Passkey](/docs/v1/workflows/authentication).
+
+### Configure External Protocol
+
+Check your appliction config in the admin console for your `APPLICATION_CLIENT_ID`.
+
+The `REDIRECT_URI` is your application's App Scheme or Universal URL.
+
+<PkceInfo />
+
+<StateInfo />
+
```

Comment:
Since we don't need to have them craft an auth URL and you listed everything they need to do under application creation I think you can remove the `Configure External Protocol` section too.

Reply:
Done. The comment did not get lost, but GitHub keeps hiding a few conversations. I have to keep selecting "Load more..." whenever I refresh the page 🙂

Reply:
Also, why did I add "PkceInfo" and "StateInfo" then? 😛

Reply:
Yea I wish GitHub hid the closed ones instead!

That’s a good point lol